charts/nublado2/templates/nublado_config.yaml

```diff
@@ -0,0 +1,9 @@
+kind: ConfigMap
```

Comment:
Could you use `nublado-config.yaml` for the file name so that it matches the name of the created object?

Reply:
Okay sure, although I'd like to figure out what we want to name this, maybe if we want to be all consistent this should just be config?

Reply:
It's going to end up in the same namespace as a bunch of JupyterHub stuff, so I like your naming of `nublado-config` for the Kubernetes resource. I think that will disambiguate. So in Kubernetes it would be the `nublado-config` ConfigMap resource in the `nublado2` namespace, and in `values.yaml` it would be `nublado.config`.